pkg/services/thumbs/crawler.go

```diff
@@ -247,3 +237,56 @@ func (r *simpleCrawler) walk() {
 	r.status.Finished = time.Now()
 	r.broadcastStatus()
 }
+
+// TODO extract the three methods below from this file
+func (r *simpleCrawler) SaveThumbnailFromFile(tempFilePath string, dashboardID int64, dashboardUID string, theme rendering.Theme, kind models.ThumbnailKind) (int64, error) {
```

Comment:
```suggestion
func (r *simpleCrawler) saveThumbnailFromFile(tempFilePath string, dashboardID int64, dashboardUID string, theme rendering.Theme, kind models.ThumbnailKind) (int64, error) {
```

Does this need to be public?  better to keep things private (lower case) unless really necessary